Chutzpah/Node/packages/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "chutzpah",
   "dependencies": {
-    "puppeteer-core": "^2.0.0",
-    "jsdom": "^11.0.0"
+    "jsdom": "^17.0.0",
```

Comment:
I think this will break jsdom integration. It is not a simple fix, let's roll this back

Reply:
I reverted it back to 11.0.0.